### ISSUE TITLE
Version Checker make crash

### ISSUE THREAD

**Issue by Qwertyuiopwasd0** [SteveKunG/SkyBlockcatia#27]
I play skyblockcatia an minecraft crash. I think maybe version checker of skyblockcatia at line 20 and 21. here's a log: https://paste.ee/p/7xcbO (sorry for my english)

**Comment by Devviedehm** [comment]
Same here, I isolated it to the general forge config file.

REQUIRES these options to be set like this. Otherwise the mod is unable to update, and instead of continuing it just throws an unhandled error and quits instead of proceeding to the game (rest of the file and other configs within it seem to not affect it, though I did not do extensive testing, only enough to get it working for myself):

> {minecraft folder}\config\forge.cfg
```
general {
 ...
 B:disableVersionCheck=false
 ...
}

version_checking {
 ...
 B:Global=true
 ...
}
```

EDIT: To add more information, I'm running Minecraft 1.8.9 with the Forge Latest (not Forge Recommended), v11.15.1.2318 because one of the mods needed it. Also using one custom texture pack.
